### ISSUE TITLE
openbsd timing.cc coredump

### ISSUE THREAD

**Issue by jungle-boogie** [nmap/nmap#472]
OpenBSD current on nmap commit 6d8b8995b6f9d5acecd0ce98bba41fc21dfcd57c results in a coredump when doing the following:

```
 $ nmap -sT 192.168.0.8  

Starting Nmap 7.25SVN ( https://nmap.org ) at 2016-07-23 11:56 PDT
assertion "diff <= interval" failed: file "timing.cc", line 440, function "update"
Abort trap (core dumped)
```

https://github.com/nmap/nmap/blob/799048e9fc576091623f9ce5f79e26d80020f98b/timing.cc#L440

**Comment by dmiller-nmap** [comment]
Fixed in d4f698b. We left in an error statement so we can figure out why this is happening in the first place, but you're right that it shouldn't cause a complete crash. Thanks!

**Comment by dmiller-nmap** [comment]
@jungle-boogie Apparently we fixed the wrong thing (which was also a reported bug), but this issue has actually been fixed in c1042454a and 6352cb1.

**Comment by jungle-boogie** [comment]
@dmiller-nmap,

YES! this actually scans without a crash!!

```
nmap -sT 192.168.0.1

Starting Nmap 7.30SVN ( https://nmap.org ) at 2016-10-19 08:26 PDT
Nmap scan report for 192.168.0.1
Host is up (0.011s latency).
Not shown: 995 filtered ports
PORT     STATE  SERVICE
22/tcp   closed ssh
23/tcp   closed telnet
80/tcp   open   http
443/tcp  open   https
1900/tcp closed upnp

Nmap done: 1 IP address (1 host up) scanned in 5.35 seconds
```
